src/integrations/analytics.ts

```diff
@@ -12,8 +12,8 @@ export async function sendAnalyticsEvent<
 >(events: T[]): Promise<void> {
   const now = new Date();
   const [visit_id, session_id] = await Promise.all([
-    generateLongId(),
-    generateLongId(),
+    generateUUID(),
+    generateUUID(),
```

Comment:
Why are we changing the format from string id to uuid?

Reply:
Reduce the chances for clashes, UUID should be more failsafe in this case and it's not user facing